akka-docs/src/main/paradox/remoting-artery.md

```diff
@@ -110,9 +111,27 @@ officially supported. If you're on a Big Endian processor, such as Sparc, it is
 
 @@@
 
+## Migrating from classic remoting
 
+Artery TCP will be the default transport in Akka 2.6.0, and the @ref:[classic remoting implementation](remoting.md)
+will be deprecated.
 
-### Canonical address
+Artery has the same functionality as classic remoting and you should normally only have to change the
+configuration to switch.
+
+Enable Artery as shown in above @ref:[configuration](#configuration) with your
+@ref:[selected transport](#selecting-a-transport). `tcp` is a good start.
+
+The protocol part in the Akka `Address`, for example `"akka.tcp://actorSystemName@10.0.0.1:2552/user/actorName"`
+has changed from `akka.tcp` to `akka`. If you have configured or hardcoded any such addresses you have to change
+them to `"akka://actorSystemName@10.0.0.1:2552/user/actorName"`. `akka` is used also when TLS is enabled.
```

Comment:
We should note this in the migration guide, I remember seeing this hard coded in tests at various places for example.

Reply:
I wonder if there is a way to more seamlessly do it.

Reply:
We will start a migration guide page for 2.5 -> 2.6 and then this will be moved into that, but right now I thought it was better to embed it here.

Just ignoring or rewriting the protocol part isn't great, better with a small explicit step. A well written application will have zero such.